vegbilder-client/src/components/ImageView/SideControlBar/MoreImageInfo/MoreImageInfo.tsx

```diff
@@ -0,0 +1,399 @@
+      (res) => {
+        if (res && res.objekter.length) {
+          const kontraktsområder_: string[] = [];
+          res.objekter.forEach((obj: any) => {
+            const kontraktsområder = obj.kontraktsområder;
+            kontraktsområder.forEach((kontraktsområde: any) => {
+              kontraktsområder_.push(kontraktsområde.navn);
+            });
+            const gate = obj.gate;
+            if (gate) {
+              setGatenavn(gate.navn);
+            }
+          });
+          setKontraktsområder(kontraktsområder_);
+        }
+      }
+    );
+  };
+
+  const getKommuneAndFylke = async (latlng: ILatlng) => {
+    const response = await GetKommuneAndFylkeByLatLng(latlng);
+    setFylkesNavn(response.fylkesnavn);
+    setKommuneNavn(`${response.kommunenavn} (${response.kommunenummer})  `);
+  };
+
+  const getDetectedObjectsJson = (imagePoint: IImagePoint) => {
+    if (imagePoint.properties.DETEKTERTEOBJEKTER) {
+      const detectedObjectsJson = JSON.parse(imagePoint.properties.DETEKTERTEOBJEKTER);
+      return detectedObjectsJson;
+    }
+  };
+
+  const getNumberOfDetectedObjects = (imagePoint: IImagePoint) => {
+    const detectedObjJson = getDetectedObjectsJson(imagePoint);
+    let n = 0;
+    for (let prop in detectedObjJson) {
+      n += parseInt(detectedObjJson[prop]);
+    }
+    return n;
+  };
+
+  useEffect(() => {
+    if (imagePoint) {
+      const NordkappLatLng = { lat: 71.1652089, lng: 25.7909877 };
+      const LindesnesLatLng = { lat: 57.9825904, lng: 7.0483913 };
+      const imagePointLatlng = getImagePointLatLng(imagePoint);
+
+      if (imagePoint.properties.AAR >= 2020) {
+        setResourceStateByEgenskapAndResourceId(
+          imagePoint,
+          fartsgrenseId,
+          'Fartsgrense',
+          setFartsgrense
+        );
+        setResourceStateByEgenskapAndResourceId(imagePoint, broId, 'Navn', setBroNavn);
+        setResourceStateByEgenskapAndResourceId(imagePoint, tunnelId, 'Navn', setTunnelNavn);
+        getTrafikkMengde(imagePoint);
+        getGateNavnAndKontraktsområder(imagePoint);
+      } else {
+        setBroNavn([]);
+        setFartsgrense([]);
+        setGatenavn(undefined);
+        setTunnelNavn([]);
+        setTrafikkMengde([]);
+        setKontraktsområder([]);
+      }
+
+      if (imagePointLatlng) {
+        getKommuneAndFylke(imagePointLatlng);
+        setPosition(imagePointLatlng);
+        const kmToLindesnes = getDistanceFromLatLonInKm(imagePointLatlng, LindesnesLatLng).toFixed(
+          2
+        );
+        const kmToNordkapp = getDistanceFromLatLonInKm(imagePointLatlng, NordkappLatLng).toFixed(2);
+        setDistanceToLindesnes(kmToLindesnes);
+        setDistanceToNordkapp(kmToNordkapp);
+      }
+    }
+  }, [imagePoint]);
+
+  interface IItemGroupContainerProps {
+    Icon:
+      | OverridableComponent<SvgIconTypeMap<{}, 'svg'>>
+      | FunctionComponent<SVGProps<SVGSVGElement>>;
+    children: JSX.Element | JSX.Element[] | void[] | any;
```

Comment:
Slet litt med denne typen. Fjerner jeg any får jeg: Type 'Element | null' is not assignable to type 'Element'. Har prøvd å legge inn null | Element men det hjalp ikke

Reply:
Hmm. Funker `children?: React.ReactNode`? 🤔  Et raskt søk gav det som forslag. 

Ellers så kan du også droppe children fra Props og heller legge på type på ImageGroupContainer,

`const ItemGroupContainer: React.FC<IItemGroupContainerProps) 
> = ({ children, Icon, headline ) => {...}`

Reply:
Ah, det funket ja :D Takk 🥳